src/components/InvoicesList/InvoicesList.js

```diff
@@ -99,8 +102,11 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
                           <td>{invoice.accountId}</td>
                           <td>{invoice.product}</td>
                           <td>
-                            <FormattedDate value={invoice.date} />
+                            {!!invoice.creationDate && (
+                              <FormattedDate value={invoice.creationDate} />
+                            )}
```

Comment:
Creo que si aplicás el cambio en el servicio(https://github.com/FromDoppler/doppler-webapp/pull/1111/files#r505662416) podrías evitar la condición:

```suggestion
                            <FormattedDate value={invoice.creationDate} />
```

Reply:
Exacto, aplicando la logica de arriba esto no tendria sentido, ya lo cambio

Reply:
Done